src/Item.php

```diff
@@ -80,6 +87,11 @@ public function getRuleName(): ?string
         return $this->ruleName;
     }
 
+    public function hasRuleName(): bool
```

Comment:
I don't think that deserves a method. `->getRuleName() !== null` should be enough.

Reply:
fixed